thirdparty/PUI_install.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+
+script_root = os.path.dirname(os.path.realpath(__file__))
+pui_folder = os.path.join(script_root, "PUI", "PUI")
```

Comment:
Using the same way in `modmesh/view.py` to check if the submodule is initialized or not.